util/Nginx/entrypoint.sh

```diff
@@ -30,7 +30,7 @@ mkhomedir_helper $USERNAME
 # The rest...
 
 chown -R $USERNAME:$GROUPNAME /etc/bitwarden
-cp /etc/bitwarden/nginx/default.conf /etc/nginx/conf.d/default.conf
+cp /etc/bitwarden/nginx/*.conf /etc/nginx/conf.d/
```

Comment:
@sevensixseven , @joseph-flinn , do you know if this will break any of our current deployment models. Not sure about expansion of `.conf` files in a VPS or other web services scenarios that the submitter has mentioned.

Reply:
@cscharf It shouldn't, but I can do a quick test.

Reply:
@cscharf Looks good.